core/src/main/scala/akka/kafka/internal/ConsumerStage.scala

```diff
@@ -190,7 +192,8 @@ private[kafka] object ConsumerStage {
       if (offsets.isEmpty)
         Future.successful(Done)
       else {
-        stages.head._2.commit(this)
+        implicit val ec = akka.dispatch.ExecutionContexts.global()
+        Future.sequence(stages.map(_._2.commit(this))).map(_ => Done)
```

Comment:
Sure, but I had to close this PR because by mistake I made it from the master branch. Before posting a new one I'll create the issue and post the proposed solution there.